applications/serial_lte_modem/doc/HTTPC_AT_commands.rst

```diff
@@ -245,13 +245,11 @@ Syntax
 
 ::
 
-   #XHTTPCRSP=<byte_received>,<state><CR><LF><response>
+   <response><CR><LF>#XHTTPCRSP:<received_byte_count>,<state>
 
-* The ``<byte_received>`` is an integer.
+* ``<response>`` is the raw data of the HTTP response, including headers and body.
+* ``<received_byte_count>`` is an integer.
   It represents the length of a partially received HTTP response.
-* The ``<state>`` value can return one of the following:
-
+* ``<state>`` is one of the following:
```

Comment:
Needs an empty line after this line.

Is there a reason for the wording change?

Reply:
The same wording is on Line 156, is the difference that there it's a response, and this is part of a notification?

Reply:
If you could change the other line as well, then. It's preferable to have the same wording when describing equivalent things, especially on the same page.